### ISSUE TITLE
Extend irrd/server/http/server.py to allow object creation by HTTP POST request

### ISSUE THREAD

**Issue by renatoalmeidaoliveira** [irrdnet/irrd#418]
**Is your feature request related to a problem? Please describe.**

An HTTP API might simplify object creating and allow a simpler implementation of a pretty frontend like RADB.
 
**Describe the solution you'd like**

It could be done by the implementation of the do_POST method, that calls  ChangeSubmissionHandler(data) with some data lint.

**Comment by mxsasha** [comment]
An API for object creation is already planned for a current development project. Do you have a particular type of interface or particular features in mind that would help this work well for you?

(It will probably not end up in the place you suggest, as the HTTP handling will probably change further in #398)

**Comment by renatoalmeidaoliveira** [comment]
At the current versions the object creation is done by file input (submit_changes.py) , or email (submit_email) an API for object manipulation ( creation, changes and exclusion) would be a great start, in the current version it could be done by interfacing ChangeSubmissionHandler() but the object submit resport returns an string for terminal printing, it would be nice some dict, and later implementing that as an REST API that would separate the IRRd server from the user application.
About the API does the project intent to use some web server framework like Flask ou something, or will keep the low level http modules?

**Comment by mxsasha** [comment]
This is now implemented: https://irrd.readthedocs.io/en/latest/users/database-changes.html

> About the API does the project intent to use some web server framework like Flask ou something, or will keep the low level http modules?

The very basic modules were really only intended for the simple status page. The current HTTP implementation uses Starlette as an ASGI web framework, on top of a uvicorn ASGI server. This choice fits particularly well with Ariadne, the GraphQL framework, but also supports other HTTP services well. I did look into FastAPI, which is designed on top of Starlette to support REST API's, but found it overkill for IRRd with only two actual API calls (plus GraphQL).
